pegasus/sites.v3/code.org/public/advocacy/sample.haml

```diff
@@ -0,0 +1,18 @@
+---
+title: Full Team
+chart: true
+layout: page_print
+theme: pdf
```

Comment:
would it make sense to (eventually) have the pdf_version=1 param automatically set these properties?